tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -51,3 +51,23 @@ Feature: full text search
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: search files using a deleted tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
+    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+      | tag1 |
+    When user "Alice" removes the following tags for file "file1.txt" of space "Personal":
+      | tag1 |
+    And user "Alice" searches for "Tags:tag1" using the WebDAV API
```

Comment:
i think make a remove tag api tests if there is not and if ts already in there than shift it to given step.